creator-node/src/TranscodingQueue.js

```diff
@@ -225,16 +225,16 @@ class TranscodingQueue {
    * The max number of transcode jobs that can run at a given moment is correlated to
    * the number of cores available.
    *
-   * If the number of active jobs is equal to the number of cores, and the number of
-   * waiting jobs is equal to or greater than the number of cores, mark the
+   * If the number of active jobs is less than MAX_ACTIVE_JOBS, and the number of
+   * waiting jobs is less than or equal to or greater MAX_WAITING_JOBS, mark the
    * TranscodingQueue as unavailable.
    *
    * @returns boolean flag if the transcode queue can accept more jobs
    */
   async isAvailable() {
     const { active, waiting } = await this.getTranscodeQueueJobs()
 
-    return !(active === MAX_ACTIVE_JOBS && waiting > MAX_WAITING_JOBS)
+    return active < MAX_ACTIVE_JOBS || waiting <= MAX_WAITING_JOBS
```

Comment:
why use `<` and `<=` -- lets pick?

Reply:
assuming the number of cores is at 8, my thinking was that if active jobs was at 8 (max jobs), we need to then check the waiting jobs to see if it is greater than 8 (basically over double)